service_storagefacility/app/services/StorageNodeService.scala

```diff
@@ -607,6 +608,8 @@ class StorageNodeService @Inject()(
             registeredDate = e.registeredDate.get,
             doneBy = e.doneBy,
             doneDate = e.doneDate,
+            id = e.affectedThing.get,
```

Comment:
This get is OK since the `affectedThing` is a required field in the database in this context.